README.md

```diff
@@ -153,3 +153,6 @@ class MyComponent extends Component {
 }
 ```
 
+## Testing using React Test Utilities
+
+For more robust example take a look at [test/react_test_utils_test.dart](test).
```

Comment:
I'd like to expand this more as a user looking at this README for the first time may not have any idea what TestUtils is or why they would want to use it.  We should have a quick summary of what they would use it for and links out to the React pages describing what TestUtils is and does.